File: punto1.py
import numpy as np

v = np.zeros(30, dtype=int)
for i in range (5000):
        candi = np.random.randint(0,30)
        v[candi] = v[candi] +1
        
voticosordenados = np.sort(v)[::-1]

for i, voticos in enumerate(voticosordenados):
    print(f"Candidato {i+1}: {voticos} votos")

File: punto1.php
<?php
$v = array_fill(0, 30, 0);
for ($i = 0; $i < 5000; $i++) {
    $candi = rand(0, 29);
    $v[$candi]++;
}

rsort($v);

echo "El orden de votos del mayor al menor son: <br>";
foreach ($v as $i => $voticos) {
    echo "Candidato " . ($i + 1) . ": " . $voticos . " votos <br>";
}
?>
